quickstarts/microsoft.dbformysql/create-flexible-server-public-access/azuredeploy.parameters.json

```diff
@@ -0,0 +1,73 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "administratorLogin": {
+            "value": "GEN-UNIQUE"
+        },
+        "administratorLoginPassword": {
+            "value": "GEN-PASSWORD"
+        },
+        "location": {
```

Comment:
remove the non-GEN parameter values and use defaultValues in the template where possible.